from unittest import TestCase


class TestMath24 (TestCase):
    def test_get_operation(self):
        self.fail()
